### ISSUE TITLE
Experiencing extremely long load times

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/radicle-upstream#1975]
I took a screen recording of this, but it's way too long so I'll just mark the delays I experienced:

- 00:00 Following tab not loading any projects.
- 00:30 Projects I'm following load
- 00:33 Click on a project
- 01:25 Header of project loads
- 01:48 The rest of the project loads

I'm not sure what's going on but it's really slow!

**Comment by geigerzaehler** [comment]
I’m experiencing this too. Looks like some of the proxy requests are blocked by the protocol. I’ll take a look.

**Comment by juliendonck** [comment]
Any insight into this @geigerzaehler ?

**Comment by geigerzaehler** [comment]
Not yet unfortunately.

**Comment by rudolfs** [comment]
This seems to be a duplicate of: https://github.com/radicle-dev/radicle-upstream/issues/1855. Can we close this?

**Comment by brandonhaslegs** [comment]
If you close this one, make sure to put #1855 in the cycle planning list, just so we don't count the task as "done".

**Comment by rudolfs** [comment]
Ok, will leave open both for now.
